### PR TITLE
[RSDK-4448] Flaky test - TestCartographerIntegration2D

### PR DESCRIPTION
**Pull request by PStrutz**
Hot fix for integration test

### CODE REVIEW

File: integration_test.go [hunk 1]
```diff
@@ -134,10 +147,11 @@ func testHelperCartographer(
 	}
 
 	done := make(chan struct{})
```
Comment (kim-mishra) [inline integration_test.go:149]:
```suggestion
	lidarDone := make(chan struct{})
```

File: integration_test.go [hunk 2]
```diff
@@ -134,10 +147,11 @@ func testHelperCartographer(
...
+	imuDone := make(chan struct{})
 	sensorReadingInterval := time.Millisecond * 200
 	timedLidar, err := testhelper.IntegrationTimedLidarSensor(t, attrCfg.Camera["name"], replaySensor, sensorReadingInterval, done)
```
Comment (kim-mishra) [inline integration_test.go:152]:
```suggestion
	timedLidar, err := testhelper.IntegrationTimedLidarSensor(t, attrCfg.Camera["name"], replaySensor, sensorReadingInterval, lidarDone)
```

File: integration_test.go [hunk 3]
```diff
@@ -157,16 +171,17 @@ func testHelperCartographer(
 	if !utils.SelectContextOrWaitChan(ctx, done) {
 		test.That(t, errors.New("test timeout"), test.ShouldBeNil)
 	}
```
Comment (kim-mishra) [inline integration_test.go:173]: Should we wait for `imuDone` here as well?

Comment (PStrutz) [reply integration_test.go:173]: No, since we are currently never passing done to the imuDone channel

Comment (kim-mishra) [reply integration_test.go:173]: cool this is a nit but can we move the comment above the `selectContextOrWaitChan`. I think comments typically refer to what is below it

**Review by kim-mishra — APPROVED** [review]
LGTM, I think it would be useful to add a comment somewhere that says `IntegrationTimedIMUSensor` will not be used for now also because I would have expected it to have gone through the IMU readings listed out. thanks for the fix!

**Pull request merged**